gpflow/utilities/utilities.py

```diff
@@ -77,6 +77,15 @@ def print_summary(module: tf.Module, fmt: str = None):
     Prints a summary of the parameters and variables contained in a tf.Module.
     """
 
+    if fmt == "notebook":
+        from IPython.core.display import display, HTML
+        tab = _print_summary_output_string(module, "html")
+        display(HTML(tab))
+    else:
+        print(_print_summary_output_string(module, fmt))
+
+
+def _print_summary_output_string(module: tf.Module, fmt: str = None) -> str:
```

Comment:
```suggestion
def tabulate_module_summary(module: tf.Module, tablefmt: str = None) -> str:
```
might be a more appropriate name, it doesn't actually print anything, the (table)fmt is passed straight through to `tabulate`, and it takes a module as an input? There might actually be use-cases in which you want this, instead of having it printed, so I think it ought to be fine as a public function.